cmd/provider/main.go

```diff
@@ -72,6 +72,14 @@ func main() {
 		ctrl.SetLogger(zl)
 	}
 
+	// configure the jitter to be the 10% of the poll interval
```

Comment:
NIT: some other providers I checked all have it set to 5%. Unless there is any reason not to, it would be cool to have it here as well. Saying that, I am not sure why 5% was chosen instead of some other number.

Reply:
sure. I picked 10% based on [provider-terraform](https://github.com/upbound/provider-terraform/pull/204) change, but without exposing it as a configurable flag option. Also, based on the observation on a cluster that 10% jitter leads to noticeable reconcile intervals spread (the graph attached in description).

if we think this needs to be made configurable through flag option, i can add that.

Reply:
If you could add it as a flag, that would be awesome. The default can be 10%

Reply:
added `--poll-jitter-percentage` flag.